docs/setup-cluster/slurm/singularity.rst

```diff
@@ -248,3 +248,48 @@ You can view the current set of Docker image names in the cache with the ``-l``
 .. code:: bash
 
    manage-enroot-cache -s /shared/enroot -l
+
+.. _singularity-apptainer-shells:
+
+******************************
+ Singularity/Apptainer Shells
+******************************
+
+When using Determined AI with Singularity/Apptainer for interactive shells, there are some important
+behaviors to understand, particularly regarding home directories and file locations.
+
+Home Directory Binding
+======================
+
+Singularity/Apptainer automatically binds the user's home directory to the container. This means:
+
+-  When you start a shell, you will be placed in your actual system home directory, not in the
+   working directory where files are copied.
+-  This behavior differs from Docker containers and can be confusing if you're expecting to see your
+   files immediately.
+
+File Locations
```

Comment:
This shouldn't be part of singularity/apptainer because this applies to a variety of `det shell start` scenarios.